constraint/pkg/regorewriter/sources.go

```diff
@@ -91,18 +91,19 @@ func (s *Sources) Reparent(old, new string) error {
 
 // Write will write the sources to the filesystem.
 func (s *Sources) Write() error {
-	return s.forAll(func(module sourceFile) error {
+	// TODO: Determine if _ is intended to be used in anything.
```

Comment:
open TODO here

Reply:
I talked with Max about this - we weren't sure a great way to mark that this is obviously a bug in the code. It isn't worth the time to fix it and verify the original intent as it was written two years ago, is untested, and isn't used in any production code.

The intent is to leave a note for someone in the future (who presumably will care about the functionality of that code) that there is obviously a problem here, but we don't really know for sure what will fix it.